corehq/apps/userreports/views.py

```diff
@@ -1326,6 +1340,40 @@ def rebuild_data_source(request, domain, config_id):
     ))
 
 
+def _number_of_records_to_be_iterated_for_rebuild(config):
```

Comment:
Would you be able to add a unit test for this function?

Reply:
Cool, let me try

Reply:
@zandre-eng added tests in https://github.com/dimagi/commcare-hq/pull/34397/commits/bbe096c406b66d58d1fff8700e0fe6ca363602b7

Added for the view itself since they were missing